ui/src/kapacitor/components/AlertTabs.js

```diff
@@ -76,18 +80,33 @@ class AlertTabs extends Component {
           this.refreshKapacitorConfig(this.props.kapacitor)
           this.props.addFlashMessage({
             type: 'success',
-            text: `Alert for ${section} successfully saved`,
+            text: `Alert configuration for ${section} successfully saved.`,
           })
         })
         .catch(() => {
           this.props.addFlashMessage({
             type: 'error',
-            text: 'There was an error saving the kapacitor config',
+            text: `There was an error saving the alert configuration for ${section}.`,
           })
         })
     }
   }
 
+  handleTestConfig = section => async () => {
+    try {
+      await testAlertOutput(this.props.kapacitor, section, {})
+      this.props.addFlashMessage({
+        type: 'success',
+        text: `Successfully sent an alert to ${section}.`,
```

Comment:
@jaredscheib commented that this success message might suggest that the alert message has successfully reached its destination- when in fact we are only reporting that it was _SENT OUT_ successfully. If the configuration is jibberish then its like the _address_ a user entered in the configuration is wrong, and the message will not reach its destination. Considering changing this message to "Successfully sent an alert to {section}, if [you do not receive it], check your configuration parameters. 
Also- I would really like to improve docs around alerting through chronograf. s00000......

Reply:
and to be sure:
1. i think the copy of `Send Test Alert to [x]` may also merit either revision or some kind of questionmark tooltip or page copy for clarification of what's really being tested. when i see this button, i expect that it is assuring me that the full roundtrip is successful, but what it's really testing is only our components, not the API. seems like Kapacitor would capture an error message if the API request failed, and we could relay that back to Chronograf for the error notification. if not, clarifying what is being tested, and what a "success" response means, would be helpful in my opinion, so as to not mislead the user.
1. also, the icon looks like an atomic experience. maybe a lightning bolt instead? :)